package.json

```diff
@@ -69,7 +69,7 @@
     "react-native-hw-keyboard-event": "0.0.4",
     "react-native-image-picker": "5.0.2",
     "react-native-in-app-review": "4.2.1",
-    "react-native-incall-manager": "github:cpoile/react-native-incall-manager",
+    "react-native-incall-manager": "github:cpoile/react-native-incall-manager#634c8fc",
```

Comment:
Could you remind me why we are using a fork?

Reply:
@streamer45 I'm trying to think why, it was back during the first v2 move, but I can't remember, sorry. Removing.

Reply:
Cool, one less thing to keep updated :)